mkdocs.yaml

```diff
@@ -41,11 +42,23 @@ plugins:
   - mkapi
 nav:
   - About: README.md
-  - Leaderboards: leaderboards
+  - Leaderboards:
```

Comment:
this specification is required to get "compare models" at the end (otherwise it appears at the top)